code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -370,13 +370,6 @@
 	startswith = list(/obj/item/weapon/light/tube = 14,
 					/obj/item/weapon/light/bulb = 7)
 
-/obj/item/weapon/storage/box/glowsticks
-	name = "box of mixed glowsticks"
-	icon_state = "box"
-	startswith = list(/obj/item/device/flashlight/glowstick = 1, /obj/item/device/flashlight/glowstick/red = 1,
-					/obj/item/device/flashlight/glowstick/blue = 1, /obj/item/device/flashlight/glowstick/orange = 1,
-					/obj/item/device/flashlight/glowstick/yellow = 1, /obj/item/device/flashlight/glowstick/random = 1)
```

Comment:
Probably a bad idea, but my reasoning at the time was that we didn't use it, and even couldn't spawn it in.